### ISSUE TITLE
Warning: Stateless function components cannot be given refs.

### ISSUE THREAD

**Issue by vnshgrg** [shoutem/ui#234]
```
<View styleName="flexible" style={{backgroundColor: '#f1f5f8', marginTop: 64, paddingBottom: 5}}>
  <ListView
    data={dataSource}
    renderRow={this._renderRow.bind(this)}
    renderHeader={this._renderHeader.bind(this)}
    onLoadMore = {this._getFeed.bind(this)}
    loading={this.state.loading}
   />
</View>
```
Just upgraded my project to `react-native 0.43.3` . Now, my `<ListView/>` throws this warning.
```
Warning: Stateless function components cannot be given refs. Attempts to access this ref will fail. 
Check the renderer method of 'Styled(Spinner)'.
...
```
If I remove the loading props for the `<ListView/>` then it doesn't throw the warning but I also don't want to git rid of that sweet ActivityIndicator loading spinner.

**Comment by JulianKingman** [comment]
This PR should fix it: https://github.com/shoutem/theme/pull/20

**Comment by zrumenjak** [comment]
This fix has been released in 0.20.0
